assets/sass/utilities/_overflow-wrap.scss

```
@@ -0,0 +1,25 @@
+/**
+ * Overflow wrap utility styles.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+	// See https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap
+
+	.googlesitekit-overflow-wrap-break-word {
+		overflow-wrap: break-word;
+	}
```

Maybe this should be closer-linked to the style in-question?
```suggestion

	// See https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap
	.googlesitekit-overflow-wrap-break-word {
		overflow-wrap: break-word;
	}
```

But also an explanation of why we're using this is probably better if it warrants a comment at all? Otherwise it can probably be removed; if anyone's confused about the attribute itself they can look it up straightforwardly 🙂